pkg/apis/trust/v1alpha1/conversion.go

```diff
@@ -34,6 +38,71 @@ func (src *Bundle) ConvertTo(dstRaw conversion.Hub) error {
 	return nil
 }
 
+func Convert_v1alpha1_BundleTarget_To_v1alpha2_BundleTarget(in *BundleTarget, out *trustv1alpha2.BundleTarget, scope apimachineryconversion.Scope) error {
+	if in.ConfigMap != nil {
+		out.ConfigMap = append(out.ConfigMap, trustv1alpha2.TargetKeyValue{Key: in.ConfigMap.Key})
+	}
+	if in.Secret != nil {
+		out.Secret = append(out.Secret, trustv1alpha2.TargetKeyValue{Key: in.Secret.Key})
+	}
+
+	if in.AdditionalFormats != nil {
+		appendTargetKV := func(tkv trustv1alpha2.TargetKeyValue) {
+			if out.ConfigMap != nil {
+				out.ConfigMap = append(out.ConfigMap, tkv)
+			}
+			if out.Secret != nil {
+				out.Secret = append(out.Secret, tkv)
+			}
+		}
+
+		if in.AdditionalFormats.JKS != nil {
+			targetKV := trustv1alpha2.TargetKeyValue{
+				Key:    in.AdditionalFormats.JKS.Key,
+				Format: trustv1alpha2.BundleFormatJKS,
+				JKS:    &trustv1alpha2.JKS{},
+			}
+			if err := Convert_v1alpha1_JKS_To_v1alpha2_JKS(in.AdditionalFormats.JKS, targetKV.JKS, scope); err != nil {
+				return err
+			}
+			appendTargetKV(targetKV)
+		}
+		if in.AdditionalFormats.PKCS12 != nil {
+			targetKV := trustv1alpha2.TargetKeyValue{
+				Key:    in.AdditionalFormats.PKCS12.Key,
+				Format: trustv1alpha2.BundleFormatPKCS12,
+				PKCS12: &trustv1alpha2.PKCS12{},
+			}
+			if err := Convert_v1alpha1_PKCS12_To_v1alpha2_PKCS12(in.AdditionalFormats.PKCS12, targetKV.PKCS12, scope); err != nil {
+				return err
+			}
+			appendTargetKV(targetKV)
+		}
+	}
+
+	out.NamespaceSelector = in.NamespaceSelector
+	if out.NamespaceSelector == nil {
+		// NamespaceSelector is required in v1alpha2
+		out.NamespaceSelector = &metav1.LabelSelector{}
+	}
+	return nil
+}
+
+func Convert_v1alpha1_JKS_To_v1alpha2_JKS(in *JKS, out *trustv1alpha2.JKS, _ apimachineryconversion.Scope) error {
+	out.Password = in.Password
+	return nil
+}
+
+func Convert_v1alpha1_PKCS12_To_v1alpha2_PKCS12(in *PKCS12, out *trustv1alpha2.PKCS12, _ apimachineryconversion.Scope) error {
+	out.Password = in.Password
+	out.Profile = trustv1alpha2.PKCS12Profile(in.Profile)
+	if out.Profile == "" {
+		// Default profile changed from LegacyRC2 to LegacyDES in v1alpha2
```

Comment:
suggestion: It would be really cool to create an ongoing list of changes we're making so we can more easily document it later. It's great that we have this comment here - maybe we could alos add all these kinds of comments to, say, `docs/alphav2-changes.md` and keep a list there?

Reply:
Yes, excellent suggestion. I will add this right away, but keep it simplistic for now, as I don't want to update it for each and every review suggestion. 😅 But listing the changes in behavior is excellent!

Reply:
I have added  `docs/alphav2-changes.md`. It appears useful and I hope it addresses your suggestion. PTAL!